src/ui.data-table/UTableRow.vue

```diff
@@ -277,6 +277,18 @@ function getCellContentClasses(row, key) {
   return typeof cellClasses === "function" ? cellClasses(row[key].value, row) : cellClasses;
 }
 
+function isEmptyValue(value) {
+  return value == null || (typeof value === "object" && !Object.keys(value).length);
+}
+
+function formatCellValue(value) {
+  if (isEmptyValue(value)) return HYPHEN_SYMBOL;
+
+  const nestedValue = value && typeof value === "object" && "value" in value ? value.value : value;
```

Comment:
"value" in value >>> value.hasOwnProperty('value')

Reply:
Tried that at first, got an error - "Do not access Object.prototype method 'hasOwnProperty' from target object.".
Possible solution - `Object.prototype.hasOwnProperty.call(value, "value")`